packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ZoomingScatterPlot/ZoomingScatterPlot.spec.js

```diff
@@ -37,14 +37,16 @@ describe('Component > ZoomingScatterPlot', function() {
     clientX: 50,
     clientY: 50,
     deltaY: -1,
-    preventDefault: sinon.spy()
+    preventDefault: sinon.spy(),
+    shiftKey: true
```

Comment:
Using Shift + Wheel to zoom JSON data plots is a potentially breaking change for Black Hole Hunters here.

Reply:
I've taken this out, in favour of leaving the existing mouse wheel behaviour in place for Black Hole Hunters, where the subject never needs to scroll vertically.